Source/Models/BrickAlignment.swift

```diff
@@ -6,10 +6,29 @@
 //  Copyright © 2016 Wayfair. All rights reserved.
 //
 
-/// Enum that indicates how bricks should be aligned within the same row
-public enum BrickAlignment {
+// BrickAlignment indicates how bricks are aligned horizontally and vertically
+public struct BrickAlignment {
+
+    let horizontal: BrickHorizontalAlignment
+    let vertical: BrickVerticalAlignment
+
+    public init(horizontal: BrickHorizontalAlignment, vertical: BrickVerticalAlignment) {
+        self.horizontal = horizontal
+        self.vertical = vertical
+    }
+}
+
+/// Enum that indicates how bricks should be aligned horizontally within the same row
+public enum BrickHorizontalAlignment {
     case Left
     case Right
```

Comment:
Should we rename these to `Leading` and `Trailing` instead? If we could do that and ensure that it works as expected when an app is in right-to-left language mode, that would be pretty powerful.

Alternatively, we could leave left/right as is for now and add Leading/Trailing later for RTL language/UI support.